src/ci_providers/provider_wercker.js

```diff
@@ -0,0 +1,66 @@
+function detect(envs) {
+  return envs.WERCKER_MAIN_PIPELINE_STARTED
+}
+
+function _getBuild(inputs) {
+  const { args, envs } = inputs
+  return args.build || envs.WERCKER_MAIN_PIPELINE_STARTED
+}
+
+function _getBuildURL(inputs) {
+  const { envs } = inputs
+  return envs.WERCKER_BUILD_URL || ''
+}
+
+function _getBranch(inputs) {
+  const { args, envs } = inputs
+
+  return args.branch || envs.WERCKER_GIT_BRANCH
+}
+
+// eslint-disable-next-line no-unused-vars
+function _getJob(envs) {
+  return ''
```

Comment:
This should take the `args` value of job

Reply:
We don't pass that in that at the moment.

Reply:
Huh. My bad, I thought we did. 🤦